metpy/io/upperair.py

```diff
@@ -204,16 +205,16 @@ def to_float(s):
         # Read all lines of data and append to lists only if there is some data
         for row in fobj:
             level = to_float(row[0:7])
-            values = (to_float(row[14:21]), to_float(row[21:28]), to_float(row[42:49]),
-                      to_float(row[49:56]))
+            values = (to_float(row[7:14]), to_float(row[14:21]), to_float(row[21:28]),
+                      to_float(row[42:49]), to_float(row[49:56]))
 
-            if any(np.invert(np.isnan(values))):
+            if any(np.invert(np.isnan(values[1:]))):
```

Comment:
Seems fine--this was just the one change that made me stare at it to understand why you made it.